.github/workflows/test.yml

```diff
@@ -77,3 +77,26 @@ jobs:
       - run: CI=false yarn build
         env:
           CI: true
+
+  run-dotrun:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dotrun
+        uses: canonical/install-dotrun@main
+
+      - name: Install dependencies
+        run: /snap/bin/dotrun install
```

Comment:
Do these need `/snap/bin/`? Is that something to do with the `canonical/install-dotrun@main` action? When we snap install things like maas we seem to be able to run the directly.